index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Hamma's Portflolio</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <nav class="nav-bar">
+            <p>Flex Box</p>
+            <img src="img/Enabled.png" alt="Logo">
+        </nav>
+        <header class="header">
+            <span class="header-text">I’m Hamma
+                Glad to see you!</span>
+            <p>
+                I’m a software developer! I can help you build a product , feature or website Look through some of my
+                work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.
+            </p>
+            <a href="#">LET’S CONNECT</a>
+            <div class="social-links">
+
+                <a href="#"><img src="img/icons/Linkedin_icon.png" alt=""></a>
+
+
+                <a href="#"><img src="img/icons/Vector.png" alt=""></a>
+
+
+                <a href="#"><img src="img/icons/Vector-1.png" alt=""></a>
+
+
+                <a href="#"><img src="img/icons/Vector-2.png" alt=""></a>
+
+
+            </div>
```

Comment:
- [ ] I suggest using `<ul>` to wrap social media icons instead of div and using `li` for each element and inside `<li>` add an `<a>` tag for each icon